front/app/components/UI/GoBackButton/GoBackButtonSolid.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from 'react';
+
+// components
+import { Button, Box, useBreakpoint } from '@citizenlab/cl2-component-library';
+import { ScreenReaderOnly } from 'utils/a11y';
+
+interface Props {
+  text: string;
+  iconSize?: string;
+  onClick: (event: React.MouseEvent) => void;
+}
+
+const GoBackButtonSolid = ({ text, iconSize = '26px', onClick }: Props) => {
```

Comment:
Just for context, why do we need a new go back button? Could we use reuse the existing one? Are we going to use this one anywhere else? Better to keep the UI consistent and not introduce new elements, unless really necessary.

Reply:
It replaces `front/app/containers/IdeasShow/GoBackButton.tsx`, so we're not adding a new one. The old one in `IdeasShow` was a horrible mess that was doing a 1000 different things based on different logic flags. This inverts the control by just passing it a `onClick` and letting the component that uses it deal with what should happen on click